boxtree/cost.py

```diff
@@ -408,24 +419,25 @@ def zero_cost_per_box(self, nboxes):
         """
         pass
 
-    def fmm_modeled_cost_per_box(self, traversal, level_to_order,
-                                 ndirect_sources_per_target_box,
+    def cost_per_box(self, traversal, level_to_order,
                                  calibration_params,
+                                 ndirect_sources_per_target_box=None,
                                  box_target_counts_nonchild=None):
```

Comment:
Minor and pedantic: my understanding is this indentation style does not comply with PEP 8. We can either make the first line empty after the parenthesis, or indent the following lines vertically aligned with "self". I am going to change it to the second style. See the examples in https://www.python.org/dev/peps/pep-0008/#indentation.

Otherwise everything looks good to me. I am going to merge this review, change the name and make sure the Pytential cost model is compatible as well.

Reply:
> I am going to change it to the second style.

Sounds good.

In general, I am not a huge fan of horizontal alignment, because (a) it'll all mean nothing if you use a variable-width font, and (b) if you use long names (and I like those!), then there's often not much horizontal space left to squeeze things into. As a result (and also based on what Vim does for me automatically), I often use the first style.

Nonetheless, I'm willing to tolerate any indentation that Flake8 doesn't complain about. :)